kroxylicious-integration-tests/src/test/java/io/kroxylicious/kms/service/KmsIT.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.kms.service;
+
+import java.lang.reflect.InvocationTargetException;
+import java.nio.ByteBuffer;
+import java.time.Duration;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Abstract Kms IT.
+ *
+ * @param <C> The config type
+ * @param <K> The key reference
+ * @param <E> The type of encrypted DEK
+ */
+@SuppressWarnings("java:S5960") // this is test code, it exists in the main module to facilitate its use by concrete test cases
```

Comment:
In checking what S5960 was I found https://github.com/SonarSource/sonar-java/blob/50333fc497aab5abce1cbf6adb7faa222fc0ee1a/java-checks/src/main/java/org/sonar/java/checks/AssertionsInProductionCodeCheck.java#L40 I wonder if this would be better in a test package instead of suppressing the rule?

Reply:
it is already redundant.. an earlier interaction had the file in the src/main.